### DISCUSSION TITLE
How to determine the best training epochs and loss?

### DISCUSSION THREAD

**Discussion by wangyueer97** [awslabs/gluonts#1306] [Q&A]
Hi,
I'm using deepar model to predict. The loss being optimized is negative log-likelihood, so the loss I got is negative. I'm wondering how to determine whether the value of loss is good enough? Is it  the case that the smaller the absolute value of loss is, the better the training is?

**Comment by kaijennissen** [comment]
The short answer is yes, lower values of the negative log-likelihood indicate a better fit. More details can be found [here](https://en.wikipedia.org/wiki/Likelihood_function).
